3ds2-example/frontend/components/advanced/components.js

```diff
@@ -5,7 +5,7 @@ import { getPaymentMethods, postDoPayment, postDoPaymentDetails } from "../../sh
 
 import { renderResultTemplate, attachClickHandlerForReset, parseRedirectResultToRequestData, getFlowType } from "../../shared/utils";
 
-const CLIENT_KEY = import.meta.env.ADYEN_CLIENT_KEY;
+const CLIENT_KEY = import.meta.env.VITE_ADYEN_CLIENT_KEY;
```

Comment:
Please change to `ADYEN_CLIENT_KEY` following our ADYEN_ prefix conventions for `github/adyen-examples`